datastore/lib/datastore_samples.ex

```diff
@@ -0,0 +1,50 @@
+defmodule GoogleApi.Datastore.Samples do
+
+  @moduledoc """
+  Documentation for GoogleApi.Datastore.Samples.
+  """
+
+  # [START datastore_insert]
+  @doc """
+  Create a Datastore entity of type "Task"
+
+  ## Examples
+
+      iex> GoogleApi.Datastore.Samples.create_task("Get Milk", "2% is best")
+
+  """
+  def create_task(name, description \\ "") do
+    Diplomat.Entity.new(
+      %{"description" => description},
+      "Task2",
+      name
+    ) |> Diplomat.Entity.insert
+    :ok
```

Comment:
Seems weird to just throw away any response from the insert and just return `:ok`

Reply:
I agree. The response was `[nil]`, so I didn't know what to do with it.

Reply:
If you have a better way to handle it LMK, otherwise I'll merge!

Reply:
That's definitely a bizarre response from Diplomat. My read of the source suggests that's what was sent back by the underlying RPC as the list of mutation responses. But I'm not comfortable relying on that, so I'll not advocate pattern matching on it. So yeah, uhh, LGTM or whatever.